packages/schematics/angular/migrations/update-11/fix-invalid-interpolations.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {extname, Path} from '@angular-devkit/core';
+import {Rule} from '@angular-devkit/schematics';
+
+const RE_INTERPOLATIONS: ReadonlyArray<[RegExp, string]> = [
+  // Matching an interpolation:
+  //
+  // {{((?:[^}'"]|'[^']*?'|"[^"]*?")*?)}
+  //        ^^^^^^                    match everything except a }, ", or '
+  //               ^^^^^^^^ ^^^^^^^^  or any quoted string
+
+  // Replace "{{expr}<!-- cmt -->}" with "{{ '{{' }} expr {{ '}}' }}".
+  // The former is a little-known pattern previously used to display an
+  // iterpolation literally, but that does not longer parse. The latter would
+  // still parse.
+  [
+    /{{((?:'[^']*?'|"[^"]*?"|[^}'"])*?)}<!--.*-->}/g,
+    //                                 ^^^^^^^^^^^ match a comment b/w braces
+    `{{ '{{' }}$1{{ '}}' }}`
+  ],
+
+  // Replace "{{expr}" with "{{expr}}".
```

Comment:
Hm, maybe we should do `{{expr} -> {{ '{{' }}expr{{ '}' }}` to preserve behavior